src/utils/uniqueObjects.js

```diff
@@ -0,0 +1,10 @@
+import _ from "lodash";
+
+const uniqueObjects = (campaign, channel) => {
```

Comment:
can be simplified to 
```
const uniqueObjects = (campaign, channel) => _.concat(_.uniqWith(campaign, _.isEqual), _.uniqWith(channel, _.isEqual))
```

Reply:
nice, and you can also get rid of the `return` statement

Reply:
9c48c4a2e684e8069e1118a0950ed72a162cdf26

Reply:
:+1: